src/psyclone/tests/psyGen_test.py

```diff
@@ -1,7 +1,7 @@
 # -----------------------------------------------------------------------------
 # BSD 3-Clause License
 #
-# Copyright (c) 2017-2020, Science and Technology Facilities Council.
```

Comment:
Can you revert all changes to this file?

Reply:
Done